### ISSUE TITLE
Is it possible that trace logs are dumped into a file?

### ISSUE THREAD

**Issue by crazytan** [alfredxiao/jackplay#3]
Sometimes it's not very convenient to use web UI to see the trace results. For example, JVM could be running inside container; or, there could be a bug in my code so that JVM exits very soon, but I still want to see the trace results.

**Comment by alfredxiao** [comment]
There is a parameter you can set that allows logs to be written to a file, I think it is something like logFile=/path/to/mylog.log. Bare in mind that the log file does not support rotation, file larger than 100M will be truncated and overridden from that on.

**Comment by crazytan** [comment]
I saw the option but in your README it writes "logFile is used to write debug/info/error log from the Jackplay Java agent, not the trace logs themselves." So what I want is to dump the trace log themselves to somewhere.

**Comment by alfredxiao** [comment]
You are right and mMy bad, I misunderstood the issue. That is a good idea and would require some changes to the code, I will work on it and hopefully can get it done soon. Many Thanks.

**Comment by crazytan** [comment]
No worries! Just thought it would be a nice feature to have. I've experimented with the trace feature provided by OpenJ9: it works, but it's very complicated. If we can do it in jackplay it would be much nicer.
